x/cron/spec/02_state.md

```diff
@@ -0,0 +1,70 @@
+<!--
+order: 2
+-->
+
+# State
+
+## State Objects
+
+The `x/cron` module keeps the following object in the state: WhitelistedContract.
+
+This object is used to store the state of a
+
+- `WhitelistedContract` - to store the contract which is whitelisted according to game type.
+
+```go
+// this defines the details of the contract registered for cronjob
+message WhitelistedContract {
```

Comment:
if you take my approach as described at the top, the specs will need to be updated